code/panjabi/index.ts

```diff
@@ -0,0 +1,31 @@
+﻿import { build, transform } from '~/base'
+
+
+const vowelDiacritics = {
+  
+    ਕ  : '[ə]',
+    ਕਾ  : '[aː]~[äː]',
+    ਕਿ  : '[ɪ]',
+    ਕੀ  : '[iː]',    
+    ਕੁ  : '[ʊ]',
+    ਕੂ  : '[uː]',
+    ਕੇ  : '[eː]',
+    ਕੈ  : '[ɛː]~[əi]',
+    ਕੋ  : '[oː]',
+    ਕੌ  : '[ɔː]~[əu]',
+
+
+}
+
+
```

Comment:
Did you run `pnpm install`? If so, it will autoformat the code and clean up this whitespace for you, if you are using VSCode.